doc/examples/reconst_bingham.py

```diff
@@ -0,0 +1,315 @@
+"""
+===========================================================================
+Reconstruction of Bingham Functions from ODFs
+===========================================================================
+
+This example shows how to reconstruct Bingham functions from orientation
+distribution functions (ODFs). Reconstructed Bingham functions can be
+useful to quantify properties from ODFs such as fiber dispersion
+:footcite:p:`Riffert2014`,:footcite:p:`NetoHenriques2018`.
+
+To begin, let us import the relevant functions and load a data consisting
+of 10 b0s and 150 non-b0s with a b-value of 2000s/mm2.
+"""
+from dipy.core.gradients import gradient_table
+from dipy.data import get_fnames
+from dipy.io.gradients import read_bvals_bvecs
+from dipy.io.image import load_nifti
+from dipy.reconst.csdeconv import (auto_response_ssst,
+                                   ConstrainedSphericalDeconvModel)
+from dipy.direction.bingham import sf_to_bingham, sh_to_bingham
+from dipy.viz import window, actor
+from dipy.viz.plotting import image_mosaic
+from dipy.core.sphere import unit_icosahedron
+
+
+hardi_fname, hardi_bval_fname, hardi_bvec_fname = get_fnames(name='stanford_hardi')
+data, affine = load_nifti(hardi_fname)
+
+bvals, bvecs = read_bvals_bvecs(hardi_bval_fname, hardi_bvec_fname)
+gtab = gradient_table(bvals, bvecs=bvecs)
+
+###############################################################################
+# To properly fit Bingham functions, we recommend the use of a larger number of
+# directions to sample the ODFs. For this, we load a `sphere` object with 12
+# vertices sampling a 3D sphere (the icosahedron). We further subdivide the
+# faces of this `sphere` representation into 5, to get 10242 directions.
+
+sphere = unit_icosahedron.subdivide(n=5)
+
+nd = sphere.vertices.shape[0]
+print('The number of directions on the sphere is {}'.format(nd))
+
+###############################################################################
+# Step 1. ODF estimation
+# =============================================================================
```

Comment:
I mean, we follow the rst rules: https://docutils.sourceforge.io/docs/user/rst/quickstart.html#sections

so, it should be this (underline same size as the title):
```rst
###############################################################################
# Step 1. ODF estimation
# ======================
```

Can you update everywhere

Reply:
Alright done!